TLM/TLM/UI/Localization/Translation.cs

```diff
@@ -189,11 +169,32 @@ public class Translation {
             return filename;
         }
 
+        private static string FixLanguageNameIfLanguageMod(string language) {
```

Comment:
Would it be better called `GetValidLanguageCode`?

Also, check to see if `language` is recognised first and if so just return it, otherwise do the `switch` to fix known weird language codes with a `default` of `en`?

Reply:
Edited comment above.

Reply:
I'm not so good at naming methods 😄 
I will refactor this to cover these cases.